store-api-server/src/user/service/dipdup.ts

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios'
+export async function getFromDipdup(walletAddress: string) {
+  const axiosTokenMetadataResponse = (await axios({
+    url: `http://${process.env['HASURA_URL']}:8080/v1/graphql`,
```

Comment:
This is probably easiest to add to GCP Secrets and import through the cloud-build.yaml